src/components/flowchart/index.js

```diff
@@ -35,8 +40,11 @@ export class FlowChart extends Component {
     this.nodesRef = React.createRef();
 
     this.handleWindowResize = this.handleWindowResize.bind(this);
+    this.handleNodeClick = this.handleNodeClick.bind(this);
+    this.handleChartClick = this.handleChartClick.bind(this);
     this.handleNodeMouseOver = this.handleNodeMouseOver.bind(this);
     this.handleNodeMouseOut = this.handleNodeMouseOut.bind(this);
+    this.handleNodeKeyDown = this.handleNodeKeyDown.bind(this);
   }
```

Comment:
Have you considered using something like https://babeljs.io/docs/en/babel-plugin-proposal-class-properties to use arrow functions instead of having to bind context in the constructor? Would change

```
this.handleNodeKeyDown = this.handleNodeKeyDown.bind(this)
```

to

```
class FlowChart extends Component {
  constructor() {
    // no need for bind
  }

  handleNodeKeyDown = () => {
   // woohoo bound context
  }
}
```

Reply:
Not too fussed either way, but it works out of the box in the current version of react-scripts, so, sure why not 🤷‍♂

Reply:
Nice, cleans it up loads, allows you to focus on important aspects of the constructor instead of the cruft 👍